### ISSUE TITLE
NTR: Mumps virus IgM testing positive in blood (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8101]
New term request Mumps virus IgM testing positive in blood (nadea3_190513003051): Mumps serology; serologies

**Comment by pnrobinson** [comment]
@MickeySegal
Note I think we want to have modifiers to indicate whether IgG or IgM etc antibodies are demonstrated for the various pathogens

Adding new term: 
- ID: HP:6000903
- label: Anti-mumps antibody positivity
- parents: [Positive bloodstream virus-specific antibody positivity (HP:0430059)](https://hpo.jax.org/app/browse/term/HP:0430059)
- pmid: 
- def: The presence of antibodies in the blood circulation that react against a component of Mumps virus.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8101
- synonyms:
